### ISSUE TITLE
terraform get step needed, not in documentation

### ISSUE THREAD

**Issue by colhom** [coreos/tectonic-installer#660]
```sh
$> terraform plan -var-file=./build/chom-test-metal/terraform.tfvars platforms/metal
Failed to load root config module: Error loading modules: module bootkube: not found, may need to be downloaded using 'terraform get'
```
In order to make this work, you have to do `terraform get ./platforms/metal` before running `terraform plan`. We should put this in the documentation.

**Comment by colhom** [comment]
I have verified this is the case with `platforms/aws` as well, I'm guessing we need to do this for all platforms.

**Comment by robszumski** [comment]
[AWS contains this](https://github.com/coreos/tectonic-installer/blob/master/Documentation/install/aws/aws-terraform.md#initialize-and-configure-terraform), can you add the same language to metal?

**Comment by zbwright** [comment]
@robszumski what about azure and openstack?

$ terraform get platforms/azure
$ terraform get platforms/openstack

?

**Comment by zbwright** [comment]
https://github.com/coreos/tectonic-installer/pull/669

**Comment by s-urbaniak** [comment]
fixed in #669, hence closing
